### ISSUE TITLE
Enhancement: Add more info to webserver (Predicted BG, Temp Basal, Raw BG, Loop Status, Uploader battery)

### ISSUE THREAD

**Issue by bmammon** [SpikeApp/Spike#158]
The newest version of Glance will allow users to see more info on their FitBit Versa: Pre-release version: https://gam.fitbit.com/gallery/clock/f89ff99d-4365-4a39-aa2f-f66aaa970587

![screen shot 2019-01-21 at 4 27 39 pm](https://user-images.githubusercontent.com/26399567/51505124-9778f800-1d99-11e9-8e0b-24fbe62ab1dc.png)

Currently, this info can only be displayed when using Nightscout as the data source. Would it be possible to add this info to Spike's webserver so we can use Spike as the data source? 

- Predicted BG
- Current BG
- Temp Basal
- Raw BG
- Loop status
- Uploader battery

![screen shot 2019-01-21 at 4 28 46 pm](https://user-images.githubusercontent.com/26399567/51505131-a790d780-1d99-11e9-9a7e-ce7f3d1dc291.png)

**Comment by miguelkennedy** [comment]
Yes... will try to have it ready for the 3.5.3 update... Will contact Ryan Mason directly to coordinate integration.

**Comment by bmammon** [comment]
Amazing! Thank you both!

**Comment by bmammon** [comment]
@miguelkennedy I also wanted to let you know that the in-app instructions for FitBit integration are outdated. There are more watchfaces available and most of them do not require the user to input the local address anymore. To install you simply need to click the gam-link which will redirect to the install link in the FitBit app. Then select Spike as the data source. 

Here is a list of all the current FitBit watchface gam-links and their associated githubs.

**Glance** - https://gam.fitbit.com/gallery/clock/7b5d9822-7e8e-41f9-a2a7-e823548c001c

https://github.com/Rytiggy/Glance/blob/master/README.md

**FlashCGM** - https://gam.fitbit.com/gallery/clock/a18fd61d-68ca-4028-af26-7ff531757715

https://github.com/PedanticAvenger/FlashCGM/wiki

**Marclock CGM & Weather** - https://gam.fitbit.com/gallery/clock/9eacf714-5b23-40c8-9621-ded74bd9edf9

https://github.com/cramis1/Marclock-with-CGM-weather

**Simple Glance** - https://gam.fitbit.com/gallery/clock/60548bf4-dd58-4268-af14-ba9322ae1a1f

https://github.com/stackenblochen/SimpleGlance/blob/master/README.md

**Orbits NS** - https://gam.fitbit.com/gallery/clock/44de5c81-b77c-4f90-baa8-38f3e3d28695

https://github.com/rajid/Fitbit-Orbits-NS

**Analog CGM** - https://gam.fitbit.com/gallery/clock/4d7b46b1-aaba-49b4-aa10-183321014dd3

https://github.com/rajid/Orbits-NS-Analog-CGM-Radial-CGM

**Radial CGM** - https://gam.fitbit.com/gallery/clock/0173730e-5381-4495-bc6e-6ec93c8df029

https://github.com/rajid/Orbits-NS-Analog-CGM-Radial-CGM

![0cffb4d4-428a-4dbc-a308-d7125d07b6e5](https://user-images.githubusercontent.com/26399567/51530262-64f9ea00-1def-11e9-9e3e-7ac0db02b6fe.jpeg)

**Comment by bmammon** [comment]
It would be useful to also have these links in the Wiki. People in the Facebook group frequently ask about FitBit watchfaces and it would be easier to refer them to the wiki

**Comment by m-a-v** [comment]
We started a FAQ/Documentation page. We could also integrate it there. I would like to discuss this with Miguel first. Currently, he seems to have less time. I'm still waiting for him to contact me :-)

**Comment by bmammon** [comment]
@miguelkennedy One more to add to the collection!

**Exerclock** - https://gam.fitbit.com/gallery/app/f483f7c4-b592-4760-a50b-234ef75cc912

https://github.com/cramis1/Exerclock
